kubernetes/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import copy
+from os.path import join
+
+class kubernetes_client_cRecipe(ConanFile):
+    name = "kubernetes_client_c"
+    version = "0.9.0"
+    package_type = "library"
+
+    # Optional metadata
+    license = "Apache-2.0"
+    url = "https://github.com/kubernetes-client/c"
+    description = "Official C client library for Kubernetes"
+    topics = ("kubernetes", "k8s", "kubernetes-client", "k8s-client")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False], "openssl_shared":[True, False], "openssl_version": ["1", "3"], "curl_version": ["7", "8"]}
```

Comment:
Don't provide the value `1` for openssl_version as it is EOL.  Even in the documentation we don't want to provide any unsafe version.

Reply:
ok, updated.